code/game/objects/items/circuitboards/circuitboard.dm

```diff
@@ -14,6 +14,7 @@
 	w_class = WEIGHT_CLASS_SMALL
 	grind_results = list(/datum/reagent/silicon = 20)
 	var/build_path = null
+	var/directional_build = FALSE //NSV13 - do we want the machine to have the frame's dir?
```

Comment:
You can technically define variables in another file with the same object. We might want to do this if we wantto make updates easier.

Reply:
From a developer standpoint I'm not a fan of split class definitions since it makes it harder to get a good overview of what you're working with.

Reply:
Thats fine it may make merge conflicts more likely though.